test/BUILD

```diff
@@ -18,6 +18,9 @@ TEST_COPTS = [
     #    "-Wshorten-64-to-32",
     "-Wfloat-equal",
     "-fstrict-aliasing",
+    ## assert() are used a lot in tests upstream, which may be optimised out leading to
```

Comment:
What "upstream" is this referring to?

I ran into the same problem with unused variables, but only needed one BENCHMARK_MAYBE_UNUSED to fix it.

Reply:
I was referring to the test code in this repo (which tends to use assert() rather than GTEST's assertion utilities)